### PR TITLE
Raiders skipjack change 3.0

### PR DESCRIPTION
**Pull request by jakmak3**
The vox raider craft, The skipjack lacks several minor items and supplies. I have added in several small items in order to rectify this and fill in the small gaps. The big gaps will be filled later (skipjack cant fly anywhere, cloner, shitty gear ect.) I have NOT added any new guns or exotic tools for the raiders OR changed what gear the raiders have on their person as soon as they spawn.

This was the skipjack before
old birdship

and this is the new 200% more swanky space bird cage
new birdship

The big things is that the bird jail is now not shitty and capable of holding prisoners along with some 02 tanks and masks to prevent slaves from dying of breathing the pure N2 air. The rest of the changes are relatively minor so i will lazily list them

Adds in 3 vox emergency lockers(the blue ones)
adds in spare vox masks and tanks in the equipment room
removes popcorn and fried eggs and putts in a spawner that spawns in 7 vox food items (rise n shiny is the most common with vox stew being the rarest)
relocates all dart guns, dart clips and ion rifles to the room the tazers are kept in (the locker in there is empty)
give the vox grey backpacks to make it easier to disguise themselves as traders (the yellow industrial backpacks just scream IM VALID!)
removes the raiders RCD (they had no use for it anyway anything it could do a welder could do)
adds in a bottle of rezadone(30u) and alkysine(10u) to deal with clones being too retarded to do anything for the rest of the round
adds in a tool locker with some welder goggles(vox suits are not welder proof)
adds some splints to deal with broken arms/legs (doesnt cover broken chest/head)
removes the n2 tanks from the airlocks since they only made it harder to move around in
puts some n2 tanks in the equipment room
skipjack brig can now have its door bolted to prevent escape
02 tanks and masks for prisoners
gives the raiders a box of handcuffs which they didnt have before
removes silicon control from all doors
all doors on the skipjack need syndicate access(the special access on all agent ID cards which the vox get)
replaces the station alert console with a camera console(on the fence about this one let me know if it should be removed)
2 gun rechargers (1 in bridge and 1 in equipment room) which they had none before
Adds in some full sized Reinforced windows to the side windows
puts some old vox suits i found in the code in inside of the raiders lockers (1 full suit of the aqua colored suit and 1 full suit of the black armor)

In later updates i will continue to change the raiders around. Refer to #13090 for the general gist of what i plan to do

remake of https://github.com/d3athrow/vgstation13/pull/13150 because github is autism

### CODE REVIEW

**Comment by jakmak3** [comment]
of fuck it compiles on my end

**Comment by Probe1** [comment]
At least you don't have 30,000 lines changed!

**Comment by jakmak3** [comment]
FUCK

**Comment by Probe1** [comment]
Downloaded a copy, compiled locally with no errors or warnings.

**Comment by Intigracy** [comment]
https://travis-ci.org/d3athrow/vgstation13/jobs/187590226

```
The command "cat maps/*.dmm | awk '/step_y =/ { exit 1 }'" exited with 1.
0.28s$ cat maps/*.dmm | awk '/layer =/  { exit 1 }'
```

never use step_y, never ever ever ever ever ever ever ever ever

File: maps/defficiency.dmm [hunk 1]
```diff
@@ -8883,7 +8883,7 @@
 "doQ" = (/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "doR" = (/obj/structure/bed/chair/comfy/black{dir = 1},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "doS" = (/obj/structure/bed/chair,/turf/unsimulated/floor{icon_state = "redyellowfull"; dir = 5},/area/tdome/tdomeadmin)
-"doT" = (/obj/machinery/door/airlock/hatch,/turf/simulated/shuttle/plating/vox,/area/shuttle/vox/station)
+"doT" = (/obj/machinery/recharger{step_y = 28},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```
Comment (Intigracy) [inline maps/defficiency.dmm:8886]: right here, pixel_y

File: maps/metaclub.dmm [hunk 2]
```diff
@@ -9240,71 +9240,71 @@
...
-"dvM" = (/obj/machinery/computer/station_alert,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
+"dvM" = (/obj/machinery/computer/security,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dvN" = (/obj/machinery/computer/shuttle_control/vox,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dvO" = (/obj/machinery/computer/crew{track_special_role = "Vox Raider"},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dvP" = (/obj/machinery/door/poddoor,/turf/simulated/floor/airless,/area)
 "dvQ" = (/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dvR" = (/obj/structure/bed/chair/comfy/black{dir = 1},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
-"dvS" = (/obj/machinery/door/airlock/hatch,/turf/simulated/shuttle/plating/vox,/area/shuttle/vox/station)
+"dvS" = (/obj/machinery/recharger{step_y = 28},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```
Comment (Intigracy) [inline maps/metaclub.dmm:9249]: pixel_y

File: maps/tgstation.dmm [hunk 3]
```diff
@@ -9060,66 +9060,66 @@
...
-"dso" = (/obj/machinery/computer/station_alert,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
+"dso" = (/obj/machinery/computer/security,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dsp" = (/obj/machinery/computer/shuttle_control/vox,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dsq" = (/obj/machinery/computer/crew{track_special_role = "Vox Raider"},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dsr" = (/obj/machinery/door/poddoor,/turf/simulated/floor/airless,/area)
 "dss" = (/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "dst" = (/obj/structure/bed/chair/comfy/black{dir = 1},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
-"dsu" = (/obj/machinery/door/airlock/hatch,/turf/simulated/shuttle/plating/vox,/area/shuttle/vox/station)
+"dsu" = (/obj/machinery/recharger{step_y = 28},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```
Comment (Intigracy) [inline maps/tgstation.dmm:9069]: pixel_y

**Review by Intigracy — DISMISSED** [review]

File: maps/defficiency.dmm (hunk 1 above)
Comment (Probe1) [reply maps/defficiency.dmm:8886]: You are looking to change ```pixel_y``` not ```step_y``` is what Inti's saying.

**Comment by Probe1** [comment]
As it was explained to me, step_x and step_y are related to pixel based movement and our codebase uses tile based.  If you did this, movement would not work and people would panic.

File: maps/defficiency.dmm [hunk 4]
```diff
@@ -8883,7 +8883,7 @@
 "doQ" = (/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "doR" = (/obj/structure/bed/chair/comfy/black{dir = 1},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "doS" = (/obj/structure/bed/chair,/turf/unsimulated/floor{icon_state = "redyellowfull"; dir = 5},/area/tdome/tdomeadmin)
-"doT" = (/obj/machinery/recharger{step_y = 28},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
+"doT" = (/obj/machinery/recharger{step_y = 26},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```
Comment (Probe1) [inline maps/defficiency.dmm:8886]: 💯

**Comment by jakmak3** [comment]
HOLY SHIT IT ACTUALLY WORKED!

**Comment by Wizardcrying** [comment]
:clap:

**Comment by jakmak3** [comment]
OK does anyone else have any more small ideas not related to cloner/surgery steup, vox shutters, memes, or anything else say them now so i can put them in while its not a total mess to do so.

**Comment by icantthinkofanameritenow** [comment]
You should make the rezadone as pills. It's too easy to overdose if it's administered from a syringe. And replace the windows with reinforced plasma glass to minimize break-ins.

**Comment by Sprok0** [comment]
Rather than giving them plasma windows, just give them blast doors or shutters and a button to close them, like the ops shuttle front window.

**Comment by jakmak** [comment]
Plasma glass windows are to good for vox and I still feel shutters are to much for them. Remember these are impoverished pirates not nuke ops

Then again it does make sense for them to have some. Maybe we could consider it

**Comment by Probe1** [comment]
i like the bebop dont you tags_bb7ba7_4427716

**Pull request merged**